trimmer_decorator.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    if @nameable.correct_name.length > 10
+      @nameable.correct_name[0...10]
+    else
+      @nameable.correct_name
+    end

```

Comment:
- [ ] OPTIONAL: you can simply write the logic in one line using ternary operator:
```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```